### PR TITLE
Add tests for sendKeys modifier releasing

### PR DESCRIPTION
**Pull request by tristanlabelle**
Making sure the behavior is as expected.

### CODE REVIEW

File: Tests/WinAppDriverTests/RequestsTests.swift [hunk 1]
```diff
@@ -4,26 +4,31 @@ import TestsCommon
...
-        let children = try XCTUnwrap(app.listView.findElements(locator: .xpath("//ListItem")))
-        XCTAssert(children.count > 0)
+    override func tearDown() {
+        app = nil
     }
 
+    // func testCanGetChildElements() throws {
+    //     let children = try XCTUnwrap(app.listView.findElements(locator: .xpath("//ListItem")))
+    //     XCTAssert(children.count > 0)
+    // }
```
Comment (jeffdav) [inline Tests/WinAppDriverTests/RequestsTests.swift:30]: We keeping this?

File: Tests/WinAppDriverTests/RequestsTests.swift [hunk 2]
```diff
@@ -96,6 +101,39 @@ class RequestsTests: XCTestCase {
         try XCTAssert(!Self.hasKeyboardFocus(app.findWhatEditBox))
     }
 
+    func testSessionSendKeys_scopedModifiers() throws {
+        try app.findWhatEditBox.click()
+        try app.session.sendKeys(Keys.shift(Keys.a) + Keys.a)
+        XCTAssertEqual(try app.findWhatEditBox.text, "Aa")
```
Comment (jeffdav) [inline Tests/WinAppDriverTests/RequestsTests.swift:107]: How does the text get reset between tests?

**Review by jeffdav — APPROVED** [review]

File: Tests/WinAppDriverTests/RequestsTests.swift (hunk 1 above)
Comment (tristanlabelle) [reply Tests/WinAppDriverTests/RequestsTests.swift:30]: Oopsie

File: Tests/WinAppDriverTests/RequestsTests.swift (hunk 2 above)
Comment (tristanlabelle) [reply Tests/WinAppDriverTests/RequestsTests.swift:107]: I'm recreating the app every time now to isolate tests

File: Tests/WinAppDriverTests/RequestsTests.swift (hunk 1 above)
Comment (tristanlabelle) [reply Tests/WinAppDriverTests/RequestsTests.swift]: Fixed

**Pull request merged**